tests/snapshots/files__if_dead_code_nested-optimize.snap

```diff
@@ -12,49 +12,89 @@ expression: visualization.result
   br v3_ .b6_ .b7_;
 .b6_:
   v8_: bool = lt v0 c2_;
-  c9_: int = const 1;
```

Comment:
Is this a regression? Code seems longer

Reply:
good point, I'll investigate

Reply:
it seems be just be flaky, I think anjali will fix it with the next select pr